src/main/java/edu/hm/hafner/analysis/parser/GhsMultiParser.java

```diff
@@ -88,7 +89,7 @@ private String extractMessage(final String messageStart, final LookaheadStream l
 
     @Override
     protected boolean isLineInteresting(final String line) {
-        return line.contains("warning") || line.contains("error");
+        return line.contains("warning") || line.contains("error") || line.contains("\\*\\(col") || line.contains("\\)\\*");
```

Comment:
Maybe this is not the ideal solution because it still asserts the first the line with `col.`